wallet/src/database/dao/WalletDAO.ts

```diff
@@ -0,0 +1,39 @@
+import { ModelCtor, Sequelize } from "sequelize";
+import { CreateWalletResponse } from "../../server/protos/schema_pb";
+import { WalletModel } from "../model";
+
+export class WalletDAO {
+  private driver: Sequelize;
+  private wallet: ModelCtor<WalletModel>;
+
+  constructor(driver: Sequelize) {
+    this.driver = driver;
+    this.wallet = driver.model(WalletModel.tableName);
+  }
+
+  async createWallet({
+    telegram_from_user_id,
+    rapyd_e_wallet,
+    rapyd_e_wallet_reference,
+    wallet_type,
+  }: {
```

Comment:
We should move this to a `type` inside the `WalletModel`. See: https://github.com/aufacicenta/rapydbot/blob/feature/9-wallet-mcs/wallet/src/database/model/WalletModel.ts#L3